simulatingrisk/hawkdovevar/model.py

```diff
@@ -13,18 +17,94 @@ def set_risk_level(self):
         # generate a random risk level
         self.risk_level = self.random.randint(0, num_neighbors)
```

Comment:
`self.random` presumably comes from the base mesa class? Otherwise wondering what it refers to

Reply:
yes, mesa provides access to built-in random methods that works just like the main python library random module

It's in mentioned in their documentation best practices, it would allow using a random seed at model instantiation: https://mesa.readthedocs.io/en/stable/best-practices.html#randomization